SharedSource/UnitTests/FunctionalProgramming/EnumerableExtensionsTest.cs

```diff
@@ -182,14 +212,14 @@ public void CreateSequence_WhilePredicateEvaluatesTrue ()
       var third = new Element (3, second);
       var fourth = new Element (4, third);
 
-      IEnumerable<Element> actual = fourth.CreateSequence (e => e.Parent, e => e != first);
+      IEnumerable<Element?> actual = ((Element?)fourth).CreateSequence (e => e!.Parent, e => e != first);

```

Comment:
Those calls / tests need another go-around, sorry. There should not be a need to do these acrobatics. That's quite breaking and annoying on the user side

Reply:
but this code would actually throw if the sequence ends before e == first

Reply:
possibly. still having an issue with the element in the sequence becoming a null option when there's none involved. Dev Jour fixe mob session today for CreateSequence?